src/parsers/transformer.rs

```diff
@@ -134,28 +135,34 @@ fn array_string_to_vec(value: &str) -> Vec<String> {
         } else if !inside_word && c == ',' {
             continue;
         } else if inside_word
-            && ((word_is_quoted && c == '"' && last_char != '\\')
+            && ((word_is_quoted && c == '"' && last_char_seen != '\\')
                 || (!word_is_quoted && c == ',')
-                || !word_is_quoted && c == '}')
+                || (!word_is_quoted && c == '}'))
         {
             inside_word = false;
             word_is_quoted = false;
-            list_output.push(output);
-            output = "".to_string();
-            println!("its the end of a word");
+            let transformed = transform(rng, &current_word, sub_type, transformer, table_name);
+            word_acc = format!("{}\"{}\",", word_acc, transformed);
```

Comment:
I think .push_str would be more efficient here to append rather than creating a whole new string